apps/bfd-pipeline/bfd-pipeline-idr/load_synthetic.py

```diff
@@ -39,15 +40,18 @@
 
 
 def load_from_csv(conn: psycopg.Connection, src_folder: str) -> None:
-    for table in tables:
-        with conn.cursor() as cur:
+    with conn.cursor() as cur:
+        # timestamps will be all over the place for synthetic data,
+        # so we need to make sure that the progress tracking doesn't mess with it.
+        cur.execute("DELETE FROM idr.load_progress")
+        for table in tables:
             # Clear out any previous data
             sql_table = table["table"]
             full_table = f"cms_vdm_view_mdcr_prd.{sql_table}"
             cur.execute(f"TRUNCATE TABLE {full_table}")  # type: ignore
             file = table["csv_name"]
             _load_file(cur, src_folder, file, sql_table, full_table)
-        conn.commit()
+            conn.commit()
```

Comment:
I imagine committing after each iteration is more manageable than a singular massive commit at the end. That seems more correct, anyhow.